components/index_client/src/seq_friends.rs

```diff
@@ -49,7 +49,8 @@ async fn seq_friends_loop(
                 let _ = response_sender.send(());
             }
             SeqFriendsRequest::ResetCountdown(response_sender) => {
-                let _ = response_sender.send(seq_friends.reset_countdown());
```

Comment:
@pzmarzly :  This one is really strange. I wonder how it happened. Maybe reset_countdown() returned something in the past? It's pretty cool clippy caught this one.